public/js/components/AtomEdit/CustomEditors/StoryQuestionFields/Question.js

```diff
@@ -25,12 +28,19 @@ export class StoryQuestionsQuestion extends React.Component {
   }
 
   render () {
+    const answersCount = (this.props.fieldValue && this.props.fieldValue.answers) ? this.props.fieldValue.answers.length : 0;
```

Comment:
Hmm, so the underlying array wrapper mutates this variable. That is where I find React lacking, there is no combinator to signal state changes upstream 😩

Reply:
In a way, yes, but realistically, the underlying array wrapper is just passing it's new value to the store, which is then propagated down.